packages/react-components/react-radio/src/components/Radio/useRadioStyles.ts

```diff
@@ -162,17 +173,24 @@ const useLabelStyles = makeStyles({
  * Apply styling to the Radio slots based on the state
  */
 export const useRadioStyles_unstable = (state: RadioState) => {
+  const { label, labelPosition } = state;
```

Comment:
I don't think `label` is used anywhere? Or am I missing it?

Reply:
Removed.